libs/langgraph/langgraph/utils/pydantic.py

```diff
@@ -41,6 +45,36 @@ def create_model(
         return create_model(model_name, **v1_kwargs, **(field_definitions or {}))
 
 
+def get_update_as_tuples(input: Any, keys: Sequence[str]) -> list[tuple[str, Any]]:
```

Comment:
Note this logic isn't for Pydantic only, it covers all non-dict cases

Reply:
agreed, i think might need to move to a different file to make that clear